src/SFA.DAS.AssessorService.Data/Apply/ApplyRepository.cs

```diff
@@ -88,9 +88,11 @@ public async Task UpdateInitialStandardData(UpdateInitialStandardDataRequest sta
             using (var connection = new SqlConnection(_configuration.SqlConnectionString))
             {
                 var result = await connection.ExecuteAsync(@"UPDATE Apply
-                                                SET  ApplyData = JSON_MODIFY(JSON_MODIFY(JSON_MODIFY(ApplyData,'$.Apply.StandardReference',@ReferenceNumber),'$.Apply.StandardCode',@StandardCode),'$.Apply.StandardName',@StandardName)
+                                                SET  ApplyData = JSON_MODIFY(JSON_MODIFY(JSON_MODIFY(ApplyData,'$.Apply.StandardReference',@ReferenceNumber),'$.Apply.StandardCode',@StandardCode),'$.Apply.StandardName',@StandardName),
+                                                    StandardCode = @StandardCode,
+                                                    ApplicationStatus = @ApplicationStatus
```

Comment:
Not sure why this needs to be an ugly `JSON_MODIFY(JSON_MODIFY(JSON_MODIFY(` sql statement.  Can't we do the ApplyData alterations in code and then just serialize and update the whole json in the table?  So deserialize ApplyData, set the properties, serialize, `UPDATE Apply SET ApplyData = @ApplyData`?
Would be much cleaner / easier to understand / would only need one UpdateApplyData method and the rest would be in the Application / easier to maintain.

Reply:
Does not need to be, but this is how its done in SQL. Lets create a new story to refactor how we are doing these queries.